### PR TITLE
requests by kim and prettier fixes

### PR DESCRIPTION
**Pull request by nyanofthemoon**

### CODE REVIEW

File: src/chartkit/components/Pie.js [hunk 1]
```diff
@@ -41,10 +64,17 @@ class Pie extends Component {
     }));
 
     return (
-     <PieWrapper style={this.props.style}>
-       {this.props.title ? <PieTitle>{this.props.title}</PieTitle> : null }
-       <ResponsivePie {...this.props} data={colorData} colorBy={data => data.color} />
-     </PieWrapper>
+      <PieWrapper style={this.props.style}>
+        {this.props.title ? <PieTitle>{this.props.title}</PieTitle> : null}
```
Comment (ciaranschutte) [inline src/chartkit/components/Pie.js]: Thoughts on pulling `title` and `style` out above it it with `const {style, tittle} = this.props` to clean it up a bit?

Comment (ciaranschutte) [reply src/chartkit/components/Pie.js]: @nyanofthemoon

File: src/chartkit/components/Pie.js [hunk 2]
```diff
@@ -19,17 +19,40 @@ const PieTitle = styled('div')`
   margin-bottom: 5px;
 `;
 
+const StyleTooltip = styled('Tooltip')`
```
Comment (hlminh2000) [inline src/chartkit/components/Pie.js]: I don't think the style is applying, we'll need `className` passed to the DOM element in the `Tooltip` component for it to apply. Also I think it should be `styled(Tooltip)` (`Tooltip` as component reference, not string)

File: src/chartkit/components/Pie.js (hunk 1 above)
Comment (hlminh2000) [reply src/chartkit/components/Pie.js]: I'm not opposed to neither for class components. Though I personally think functional components remove these kind of noise, but it's mainly a personal preference :P

Comment (hlminh2000) [reply src/chartkit/components/Pie.js]: I'm not opposed to neither for class components. Though I personally think functional components remove these kind of noise, but it's mainly a personal preference :P

Comment (nyanofthemoon) [reply src/chartkit/components/Pie.js]: Not opposed, just wanted to do the cleanup after plugging in the data provider; there might be additional things to cleanup.

File: src/chartkit/components/Pie.js (hunk 2 above)
Comment (nyanofthemoon) [reply src/chartkit/components/Pie.js]: @hlminh2000 Works for me using the component name as a string.

Comment (nyanofthemoon) [reply src/chartkit/components/Pie.js]: screen shot 2019-02-04 at 11 53 44 am

**Comment by hlminh2000** [comment]
oh the conflict thought... :P

**Pull request merged**